### PR TITLE
styled homepage and create navigation component

### PR DESCRIPTION
**Pull request by stellanw**

### CODE REVIEW

File: components/Logo/placeholder.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    x="0px"
+    y="0px"
+	width="36px"
+    height="36px"
+    viewBox="0 0 153 157"
+    enable-background="new 0 0 153 157"
+    xml:space="preserve"
+  >
+    <path
+      fill="#FCFCFC"
```
Comment (avoesen) [inline components/Logo/placeholder.js:19]:
```suggestion
 
```
i would clear the background for the design

File: components/Logo/placeholder.js
Comment (avoesen) [inline components/Logo/placeholder.js:1]: I would suggest outsourcing the SVG.

File: pages/_app.js [hunk 2]
```diff
@@ -1,15 +1,22 @@
 import GlobalStyle from "../styles";
 import { SWRConfig } from "swr";
+// import { BrowserRouter } from "react-router-dom";
```
Comment (avoesen) [inline pages/_app.js:3]:
```suggestion

```

File: pages/_app.js [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
 
 const fetcher = (url) => fetch(url).then((response) => response.json());
 
 export default function App({ Component, pageProps }) {
+  // if (typeof document === "undefined") {
+  //   return null;
+  // }
+
```
Comment (avoesen) [inline pages/_app.js:11]:
```suggestion
```

File: pages/_app.js [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
   return (
     <>
       <GlobalStyle />
+      {/* <BrowserRouter> */}
```
Comment (avoesen) [inline pages/_app.js:15]:
```suggestion
```

File: pages/_app.js [hunk 5]
```diff
@@ -1,15 +1,22 @@
...
       <SWRConfig value={{ fetcher }}>
         <Component {...pageProps} />
       </SWRConfig>
+      {/* </BrowserRouter> */}
```
Comment (avoesen) [inline pages/_app.js:19]:
```suggestion
```

**Review by avoesen — COMMENTED** [review]
I would adjust the SVG, but otherwise, great job! 🥳

File: pages/_app.js (hunk 5 above)
Comment (stellanw) [reply pages/_app.js]: Thanks Anna. I added all suggested changes.

**Pull request merged**